tools/getbinaries.py

```diff
@@ -93,7 +93,7 @@
 			dependencies += match.split();
 		dependencies = list(set(dependencies))
 		for x in dependencies:	
-			dependentOn[x + "port"] += 1
+			dependentOn[x + "port"] += [name]
```

Comment:
So you want to list all dependencies or the first N?

Reply:
The table was still readable with all, so I went ahead with all dependencies. :)